cmd/insert-user/generate-fake-id.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+id=$(dd if=/dev/urandom bs=1 count=32 2>/dev/null | base64 -w0)
+echo "@$id.ed25519"
```

Comment:
```suggestion
echo "@${id}.ed25519"
```

Just some bash-foo :upside_down_face:

Reply:
thanks :) definitely a novice in the bash apartment still :3